deploy/catalog_resources/courier/activemq-artemis-operatorsource.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorSource
+metadata:
+  name: activemq-artemis
+  namespace: openshift-marketplace
+spec:
+  type: appregistry
+  endpoint: https://quay.io/cnr
+  registryNamespace: rh_messaging
+  displayName: "Active AMQ Operator"
```

Comment:
@redhatHameed The displayName will need to be updated. The upstream project is 'ActiveMQ Artemis Operator', while AMQ is a downstream product name.

Reply:
fixed and update the display name as ActiveMQ Artemis Operator